tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseWebHookResource.java

```diff
@@ -0,0 +1,53 @@
+package hudson.plugins.tfs.rm;
+
+/**
+ * Implements ReleaseWebHookResource. Model to represent webHook register event.
+ * @author kasubram
+ */
+public class ReleaseWebHookResource {
+    private String operationType;
+    private String webHookName;
+    private String projectName;
+    private String payloadUrl;
+    private String secret;
+
+    public String getOperationType() {
+        return this.operationType;
+    }
+
+    public void setOperationType(final String operationType) {
+        this.operationType = operationType;
+    }
+
+    public String getWebHookName() {
```

Comment:
where do we make sure webhook name is unique?

Reply:
We don't. I don't see a need this to be unique. Its just a list, and the event will be sent to all the webhook.